apispec/api-spec.yml

```diff
@@ -33,6 +33,8 @@ paths:
     $ref: 'company-delta-spec.yml'
   /delta/company/validate:
     $ref: 'company-delta-spec.yml'
+  /delta/psc-delta-spec:
```

Comment:
This endpoint should be /delta/pscs. There should also be a validate endpoint like above that links to the same spec.